kt/godot-runtime/src/main/kotlin/godot/core/Variant.kt

```diff
@@ -1,9 +1,42 @@
 package godot.core
 
+import godot.core.VariantType.*
 import godot.util.toRealT
 import java.nio.ByteBuffer
 
 
+val variantMapper = mutableMapOf(
```

Comment:
Isn't the new `JVM_BYTE` missing?

Reply:
You are right, we forgot to add it to the allowed types.

Reply:
done